spyder_kernels/customize/spydercustomize.py

```diff
@@ -823,7 +846,7 @@ def runfile(filename, args=None, wdir=None, namespace=None, post_mortem=False):
             ipython_shell = get_ipython()
             ipython_shell.run_cell_magic('cython', '', f.read())
     else:
-        execfile(filename, namespace)
+        get_ipython().safe_execfile(filename, namespace)
```

Comment:
Wow, this is a big change! Please revert it because it has nothing to do with this PR.

Reply:
I separated that to https://github.com/spyder-ide/spyder-kernels/pull/134.

Could you have a look at this PR? If you thing these changes are good I won't need to change that here.

Reply:
I'll take a look at this again once #111 is merged.